pegasus/src/homepage.rb

```diff
@@ -31,6 +31,11 @@ def self.get_announcement_for_page(page, request)
       # If the banner has an array of languages, then the current language must be one of them.
       next if banner["languages"] && !banner["languages"].include?(request.language)
 
+      # If the banner has an showInternationally flag,
```

Comment:
Not blocking, very minor typo: `an` should be `a`.

Reply:
Thanks!